### PR TITLE
Add Public option to GCS UI, doesnt have Skip

### PR DESCRIPTION
**Pull request by royendo**
https://jam.dev/c/1e87d733-3fec-43d7-bebe-bd0433bb51ae

Yoyo, just discussed with EricO on one of our reviews and feedback from Mike on the public buckets so made a mini PR that adds the public option (changes button to Continue) into the UI. Feel free to close this and re-implement it in a different PR. 

https://www.figma.com/design/JtG3sbaopjO0xQlyeCjmho/RILL-WIP?node-id=773-90734&t=zfQRa7sVbwkPPF2z-1

**Checklist:**
- [ ] Covered by tests
- [ ] Ran it and it works as intended
- [ ] Reviewed the diff before requesting a review
- [ ] Checked for unhandled edge cases
- [ ] Linked the issues it closes
- [ ] Checked if the docs need to be updated. If so, create a separate Linear DOCS issue
- [ ] Intend to cherry-pick into the release branch
- [ ] I'm proud of this work!

### CODE REVIEW

File: web-common/src/features/sources/modal/GCSMultiStepForm.svelte [hunk 1]
```diff
@@ -63,6 +62,8 @@
               alwaysShowError
             />
           </div>
+        {:else if option.value === "public"}
+          <!-- No further context needed -->
```
Comment (lovincyrus) [inline web-common/src/features/sources/modal/GCSMultiStepForm.svelte:66]: If no further context is needed, we should remove this codeblock.

**Review by lovincyrus — CHANGES_REQUESTED** [review]
One comment, then LGTM

File: web-common/src/features/sources/modal/GCSMultiStepForm.svelte (hunk 1 above)
Comment (royendo) [reply web-common/src/features/sources/modal/GCSMultiStepForm.svelte:66]: so one reason i want to eventually remove skipi is this:

Screenshot 2025-11-25 at 11 12 38

but merging your recent code changes kind of explode this PR so i can take a look to make it work later :)

**Comment by royendo** [comment]
will re-open with a new branch
